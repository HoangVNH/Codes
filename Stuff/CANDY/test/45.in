10
99
DVDXVXDVXVXDDVVDVDVVDXDXDDVDDXDDVDXXDDVDVXXDXVVVXVVXVDVXVXVXXVDDVXDXVVVVDVDVDVDVVDDVXXDXXVXXVDXVVXX
87
DVDXVXVDDVXXDDVVDVVDDVVXVXDXXVDXVXVXXDDDDDDDDVXDVVDVVXXXXDXVDXXDDDVXVXDXVXXVVVDDDVVVXDV
55
DDDVVDVXXXVDVXXVXDDVVDDDDDXXXVVVVDDXVDDXVVXDDVDXXDVVVVV
98
VXVXDVDXXVDVXDXDXXVXVDVVVVDXVDVDXXVVXDDXVXVDDVXXVVVVXDXXVXVXDVXDVDXVDXXVDDDDXXVVVVVDVXVXXVVVDDXXXD
62
DVVDVVXXXVXVDXXVVXVVXDXVVXVDDDXXVDDDXXVXVDDXDDXVDDXVVDVXXDXVXX
81
XVVVXDDVVVVXDXDXVVDDDVVDDDXDVXVDDXDVDXXVXVVXDDDDXDDXVDVXXXDDXXXDVVXVXXDDXDVXXVDXX
100
DDDXVVXVDDXXXDDVDXXVDVXXVDXDVVVXDXVVDDVXDVXXVXXDXVDXVXVVVXDDDDVXXDVDXVDDVXXXDDVVXDXXXXDVVXVVXVXDDVXD
82
DDDVDDVXVXVXDDDDVXXXXVDDVVVDXVDVXVXXXXXDDVDVVVXXDDXVXVVXDDDXXVXDVVXDDXXVXDDVVVDDXD
79
XVXXDVXDXXDVXXXDXVVVDXVXDVDXDXXVXVDDXXXDVVDXXVVDXDDXDXVDXXVVDDVXDVVDVDDVXDDXVXV
81
VVXDVDVVDVDVDVDDVDDVDVDDXXVVVDVXXXVXDVDVVDXVVDVXVXVXDDXVXDXVVVXVDVDVDVVDDVVDXDXXX
