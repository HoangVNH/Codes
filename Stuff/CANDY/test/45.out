98
86
54
96
61
80
98
81
78
79
